vsintegration/src/FSharp.Editor/Diagnostics/UnnecessaryParenthesesDiagnosticAnalyzer.fs

```diff
@@ -70,13 +71,36 @@ type internal UnnecessaryParenthesesDiagnosticAnalyzer [<ImportingConstructor>]
                         let getLineString line =
                             sourceText.Lines[Line.toZ line].ToString()
 
-                        let! unnecessaryParentheses = UnnecessaryParentheses.getUnnecessaryParentheses getLineString parseResults.ParseTree
+                        let! unnecessaryParentheses =
+                            async {
```

Comment:
Is this `async` really needed, since we're already inside of a background `CancellableTask`?

Reply:
It doesn't seem to me like it is needed, so I went ahead and removed it in 282ea71.